### PR TITLE
Add AWS::CodeCommit::Repository

### PR DESCRIPTION
**Pull request by seanmcl**

### CODE REVIEW

File: src/main/scala/com/monsanto/arch/cloudformation/model/resource/CodeCommit.scala [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                              Name: String
+                            )
+object CodeCommitTrigger {
+  private type T = CodeCommitTrigger
+  implicit lazy val format: JsonFormat[T] = jsonFormat5(apply)
+}
+
+case class `AWS::CodeCommit::Repository`(
+                                          name: String,
+                                          RepositoryDescription: String,
```
Comment (tj-corrigan) [inline src/main/scala/com/monsanto/arch/cloudformation/model/resource/CodeCommit.scala]: per https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codecommit-repository.html this should be `Option[String]`

File: src/main/scala/com/monsanto/arch/cloudformation/model/resource/CodeCommit.scala [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                                          RepositoryName: Option[String] = None,
```
Comment (tj-corrigan) [inline src/main/scala/com/monsanto/arch/cloudformation/model/resource/CodeCommit.scala]: per https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codecommit-repository.html this should be `String`

**Comment by tj-corrigan** [comment]
I made the changes myself and merged under PR #200

**Comment by seanmcl** [comment]
Thanks!  Sorry I somehow missed your comments.

**Pull request closed without merging**